duckduckgo/iOS
# Add statistics params

Reviewer: Chris & Caine
Asana: https://app.asana.com/0/361428290920652/382789528015017

**Description**:
Adds a campaign param (`atb` value which is retrieved on first launch) and source param (`t` value which is based on app version) to searches

**Steps to test this PR**:
Watch search traffic (I usually use Charles but the Safari dev tools also allow it) and ensure that searches have an `atb` and `t` param

###### Reviewer Checklist:


###### PR DRI Checklist:
